sidecar/config/config.go

```diff
@@ -358,7 +366,10 @@ func (c *Config) Validate() error {
 	}
 
 	if c.Proxy {
-		validators = append(validators, IsInSet("Rules service backend", c.RulesBackend, []string{Amalgam8Backend, KubernetesBackend}))
+		validators = append(
+			validators,
+			IsInSet("Rules service backend", c.RulesBackend, []string{Amalgam8Backend, KubernetesBackend}),
```

Comment:
Rename Amalgam8Backend to StandaloneBackend (signifying that the controller is just a standalone instance in this case).